documentation/modules/ROOT/pages/operations/openshift.adoc

```diff
@@ -8,56 +8,75 @@
 
 toc::[]
 
-The following describes how to set up the {prodname} connectors for change data capture on Red Hat's https://www.openshift.com/[OpenShift] container platform.
+This procedure is for setting up {prodname} connectors on Red Hat's link:https://www.openshift.com/[OpenShift] container platform. These instructions have been tested with the two most recent releases of OpenShift. 
 
-These instructions have been tested using the https://github.com/minishift/minishift[Minishift tool]
--- allowing you to run a single node OpenShift instance locally on your machine.
+To get started more quickly, try the link:https://learn.openshift.com/middleware/debezium-getting-started/[{prodname} online learning scenario].
+It starts an OpenShift cluster just for you, which lets you start using {prodname} in your browser within a few minutes.
 
-You can find a complete example of this set-up using Minishift in our https://github.com/debezium/debezium-examples/tree/master/openshift[examples repository].
+== {prodname} Deployment
 
-And if you want to get started even quicker, try out the https://learn.openshift.com/middleware/debezium-getting-started/[{prodname} online learning scenario].
-It starts an OpenShift cluster just for you, allowing you to take your first steps with {prodname} in your browser just within a few minutes.
+To set up Kafka and Kafka Connect on OpenShift, use the set of images that are provided by the link:https://strimzi.io/[Strimzi] project. These images offer "Kafka as a Service" by providing enterprise grade configuration files and images that bring Kafka to OpenShift.
 
-== {prodname} Deployment
+.Prerequisites
 
-For setting up Kafka and Kafka Connect on OpenShift, a set of images provided by the https://strimzi.io/[Strimzi] project can be used, which offers "Kafka as a Service".
-It consists of enterprise grade configuration files and images that bring Kafka to OpenShift.
+* The OpenShift command line interface (`oc`) is installed.
+* Docker is installed. 
 
-First, install the operators and templates for the Kafka broker and Kafka Connect into our OpenShift project:
+.Procedure
 
+. In your OpenShift project, enter the following commands to install the operators and templates for the Kafka broker and Kafka Connect:
++
 [source,shell,subs="attributes",options="nowrap"]
 ----
 export STRIMZI_VERSION={strimzi-version}
 git clone -b $STRIMZI_VERSION https://github.com/strimzi/strimzi-kafka-operator
 cd strimzi-kafka-operator
 
-# We need to create security objects as part of installation so it is necessary to switch to admin user
+# Switch to an admin user to create security objects as part of installation:
 oc login -u system:admin
 oc create -f install/cluster-operator && oc create -f examples/templates/cluster-operator
 ----
```

Comment:
After that, a link like this would be nice: "To learn more about setting up Apache Kafka with Strimzi on Kubernetes and OpenShift, refer to the [right part of strimzi doc link]".

Reply:
This is the best URL that I could find. Let me know if there is a better one. 

https://strimzi.io/docs/operators/latest/overview.html#kafka-components_str

Reply:
👍